DIS-681: fix Open Archives indexing

Chloe070196 opened the pull request:
Jira ticket: https://aspen-discovery.atlassian.net/browse/DIS-681

Jacobomara901 commented:
Before: 
<img width="936" alt="image" src="https://github.com/user-attachments/assets/90030689-b441-46fd-a747-e99c520d96c0" />
After:
<img width="903" alt="image" src="https://github.com/user-attachments/assets/2e9ef64e-ebc8-4b57-bb6c-a84c50e6d191" />
Looks good to me.

Jacobomara901 commented:
Would it make more sense to use a varchar with a higher character limit instead? to keep the column format the same? I believe it's ever so slightly more efficient than text

Jacobomara901 commented:
Patch does as expected, in any case!

Chloe070196 committed:
DIS-681: fix: increase permanentUrl to varchar(2048)

Some Open Archives records have permanent URLs
longer than 512. This results in those records
not being indexed due to the current constraint
on the permanentUrl column. Increase the character
limit to 2048 in order to prevent such indexing
issues from occuring.

Chloe070196 committed:
DIS-681: docs: update release notes
